transpiler/src/main/java/org/jsweet/transpiler/candy/CandyProcessor.java

```diff
@@ -181,17 +181,17 @@ public void processCandies(TranspilationHandler transpilationHandler) throws IOE
 	/**
 	 * Returns true if the candy store contains the J4TS candy.
 	 */
-	public boolean isUsingJavaRuntime() {
+	public File getUsingJavaRuntime() {
 		if (candyStore == null) {
-			return false;
+			return null;
 		} else {
 			for (CandyDescriptor c : candyStore.getCandies()) {
 				if (c.name != null && c.name.equals("j4ts")) {
 					logger.info("found j4ts Java runtime in classpath");
-					return true;
+					return null; // FIXME

```

Comment:
It is unclear whether this check was ever effective, so I return null here.
If `CandyDescriptor` can provide a `File`, then we should return it here.